sky/clouds/azure.py

```diff
@@ -237,6 +239,18 @@ def make_deploy_resources_variables(
         gen_version = azure_catalog.get_gen_version_from_instance_type(
             r.instance_type)
         image_config = self._get_image_config(gen_version, r.instance_type)
+        # Setup commands to eliminate the banner and restart sshd.
+        # This script will modify /etc/ssh/sshd_config and add a bash script
+        # into .bashrc. The bash script will restart sshd if it has not been
+        # restarted, identified by a file /tmp/__restarted is existing.
+        # pylint: disable=line-too-long
+        cloud_init_setup_commands = base64.b64encode(
+            textwrap.dedent("""\
+            #cloud-config
+                runcmd:
+                - sed -i 's/#Banner none/Banner none/' /etc/ssh/sshd_config
+                - echo '\\nif [ ! -f "/tmp/__restarted" ]; then\\n  sudo systemctl restart ssh\\n  sleep 2\\n  touch /tmp/__restarted\\nfi' >> /home/azureuser/.bashrc
```

Comment:
IIRC it doesn't work if we restart the ssh service in the `runcmd` section here. And if we restart the service in .bashrc it works, so we need an indicator file to avoid restarting ssh service each time we run a command

Reply:
I'll do a test to double check this statement

Reply:
Yes, confirmed. Directly running `sudo systemctl restart ssh` doesn't work.

Reply:
update: `systemctl restart ssh` also doesn't work